src/pages/Contact/Contact.js

```diff
@@ -1,37 +1,47 @@
-/*****************************
-  TODO: Refactor to use builder data instead
-  ###
-  First console.log(data); to see how the data is structured
-  Our new data is going to live under 'builder'
-  Second lets componentize these sections
-  NOTE: These components should be placed in folder name with leading name, exampled Title Subtitle Body should live in a title directory
-  Lastly, please delete this comment
-******************************/
-
 import React from 'react';
 import Layout from 'components/Layout/Layout';
+import { ReactComponent as Instagram } from 'assets/Instagram_black.svg';
+import { ReactComponent as YouTube } from 'assets/YouTube_black.svg';
+import { ReactComponent as Facebook } from 'assets/Facebook_black.svg';
 
 const Contact = ({ data }) => {
   if (!data) return null;
 
   return (
     <Layout>
       <div className="contact">
+        <h2 className="contact__title">Contact Us</h2>
         <div className="contact__section">
           <div className="contact__col">
-          <h3>{ data.section_1.section_title }</h3>
-          <div dangerouslySetInnerHTML={{ __html: data.section_1.section_body.left }} />
+            <h3>{data.section_1.section_title}</h3>
+            <div dangerouslySetInnerHTML={{ __html: data.section_1.section_body.left }} />
+            <div className="contact__social">
+              <a href="https://YouTube.com/" target="_blank" rel="noopener noreferrer">
```

Comment:
These social links just take you to youtube.com, etc. homepage. Not actual HUOA accounts